tests/test_status_change.py

```diff
@@ -1815,7 +1815,7 @@ async def test_automerge_label_removed_by_core_dev():
             "head": {"sha": sha},
             "number": 5547,
             "title": "bpo-32720: Fixed the replacement field grammar documentation.",
-            "body": "\n\n`arg_name` and `element_index` are defined as `digit`+ instead of `integer`.\n\nAutomerge-Triggered-By: @miss-islington",
+            "body": "\n\n`arg_name` and `element_index` are defined as `digit`+ instead of `integer`.\n\nAutomerge-Triggered-By: GH:miss-islington\n\nAutomerge-Triggered-By: @miss-islington",
```

Comment:
Why is Automerge-Triggered-By: GH:miss-islington appearing twice?
Perhaps you meant just 
" ... Automerge-Triggered-By: GH:miss-islington" ?

Reply:
It's a bit of a test hack to make sure both forms get removed.  I can duplicate the test if you prefer :)

Reply:
Ok got it. I think it's ok like this. Thanks.